kalamine/msklc_manager.py

```diff
@@ -156,9 +160,10 @@ def build_msklc_dll(
             subprocess.run(
                 [kbdutools, "-u", arch_flag, klc_file], capture_output=not self._verbose
             ).check_returncode()
-            move(dll, INST_DIR / Path(arch))
+            move(str(dll), str(INST_DIR / Path(arch)))
 
         # Restore write permission
         for suffix in c_files:
             os.chmod(klc_file.with_suffix(suffix), S_IWUSR)
         os.chdir(prev)
+        return True
```

Comment:
Furtive bugfix ? :-)

Reply:
Not really, it’s to avoid scary traceback on errors that can happen. Makes a cleaner output :)